### PR TITLE
Display datahub link for datasets with defined URN

### PR DESCRIPTION
**Pull request by reesercollins**
### ADDITIONAL INFORMATION

- [x] Has associated issue: https://cccs.atlassian.net/browse/CLDN-1491
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/config.py [hunk 1]
```diff
@@ -1371,6 +1371,8 @@ def SQL_QUERY_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 # Set to False to only allow viewing own recent activity
 ENABLE_BROAD_ACTIVITY_ACCESS = True
 
+DATAHUB_URL = "https://localhost:9002/"
```
Comment (cccs-tom) [inline superset/config.py:1374]: This looks like a value for the development environment. Are we good with leaving this here? I'm guessing it's meant to be overridden?

**Review by cccs-tom — APPROVED** [review]

**Comment by cccs-rc** [comment]
Is this the kind of thing that could/should have some test coverage?

**Comment by reesercollins** [comment]
> Is this the kind of thing that could/should have some test coverage?

Probably, but it's not much use while the tests are unrunnable.

**Comment by cccs-rc** [comment]
> > Is this the kind of thing that could/should have some test coverage?
> 
> Probably, but it's not much use while the tests are unrunnable.

But one day, those are gonna be fixed, right? Right?! :)

**Comment by reesercollins** [comment]
> > > Is this the kind of thing that could/should have some test coverage?
> > 
> > 
> > Probably, but it's not much use while the tests are unrunnable.
> 
> But one day, those are gonna be fixed, right? Right?! :)

Hopefully yes, but it's kinda hard to write tests when you can't run said tests

**Comment by cccs-rc** [comment]
> > > > Is this the kind of thing that could/should have some test coverage?
> > > 
> > > 
> > > Probably, but it's not much use while the tests are unrunnable.
> > 
> > 
> > But one day, those are gonna be fixed, right? Right?! :)
> 
> Hopefully yes, but it's kinda hard to write tests when you can't run said tests

I vote write your tests against 2.0, or whatever you have to do to get them running, or raise a ticket to add these tests and do it as soon as our 2.0 branch is in good shape.

**Review by cccs-rc — APPROVED** [review]

**Pull request merged**